packages/next-on-pages/package.json

```diff
@@ -16,8 +20,8 @@
 	"scripts": {
 		"lint": "eslint src templates",
 		"types-check": "tsc --noEmit",
-		"build:types": "tsc -p tsconfig.api.json",
-		"build": "esbuild --bundle --platform=node ./src/index.ts ./src/api/index.ts --external:esbuild --external:chokidar --external:server-only --outdir=./dist",
+		"build:types": "tsc -p tsconfig.api.json -p tsconfig.fetch-handler.json",
+		"build": "esbuild --bundle --platform=node ./src/index.ts ./src/api/index.ts ./src/fetch-handler/index.ts --external:esbuild --external:chokidar --external:server-only --outdir=./dist",
```

Comment:
i dont see how a user would ever encounter this, nor am i really sure how to make esbuild output a .mjs instead of a .js. you cant specify the extension in the outdir. i'll change platform to browser, but other than that, not sure what you want me to do

Reply:
> i dont see how a user would ever encounter this

yeah... I think I've been overthinking/overengineering this 🤔👍 

> nor am i really sure how to make esbuild output a .mjs instead of a .js. you cant specify the extension in the outdir.

using `outfile` as I showed in the screenshot above? 🤔 
(anyways yeah it's likely not necessary 👍)

>  i'll change platform to browser, but other than that, not sure what you want me to do
👍

Reply:
I tried what you did in the screenshot but it didn't work for me